src/Controller/Component/AuthComponent.php

```diff
@@ -268,7 +268,7 @@ public function authCheck(EventInterface $event): ?Response
         $controller = $event->getSubject();
 
         $action = $controller->getRequest()->getParam('action');
-        if (!$controller->isAction($action)) {
+        if ($action !== null && !$controller->isAction($action)) {
```

Comment:
Should this have been
`$action === null ||`
?

Reply:
It should have 😢

Reply:
It seems like we need a proper unit test for this.